tomcat/conf/server.xml

```diff
@@ -165,6 +165,9 @@
                prefix="localhost_access_log" suffix=".txt"
                pattern="%h %l %u %t &quot;%r&quot; %s %b" />
 
+        <!-- Detect if Fedora is running behind a reverse proxy -->
+        <Valve className="org.apache.catalina.valves.RemoteIpValve"
+        protocolHeader="X-Forwarded-Proto" />

```

Comment:
Should we add support for the additional Headers set by Caddy?
- `hostHeader="X-Forwarded-Host"`
- `remoteIpHeader="X-Forwarded-For"`

Reply:
sure, added with [075e2e2](https://github.com/fcrepo-exts/fcrepo-docker/pull/16/commits/075e2e22da48cbd40a4fa56632f353a1de9b1780).